src/components/ui/BaseCheck.vue

```diff
@@ -1,7 +1,26 @@
 <template>
-  <img :src="ellipseSvg" class="hidden lg:block lg:pr-4" />
+  <img
+    :src="currentSvg"
+    class="sm:hidden lg:block lg:pr-4"
+    @click="clickedSvg"
+  />
 </template>
 
 <script setup lang="ts">
 import ellipseSvg from "../../assets/svg/ellipse.svg";
+import ellipseCheckedSvg from "../../assets/svg/ellipse-checked.svg";
+import { ref } from "vue";
+
+const emit = defineEmits(["update:modelValue"]);
```

Comment:
why do you emit update:modelValue here?

Reply:
This will be fixed when implementing reading/writing from local storage